packages/sit-onyx/src/components/OnyxAccordion/OnyxAccordion.vue

```diff
@@ -45,7 +52,10 @@ provide(ACCORDION_INJECTION_KEY as AccordionInjectionKey, {
 
 <template>
   <div :class="['onyx-component', 'onyx-accordion', densityClass]">
-    <slot></slot>
+    <template v-if="typeof skeleton === 'number'">
+      <OnyxAccordionItem v-for="i in skeleton" :key="i" skeleton />
```

Comment:
Oh the skeleton count is defined by the items passed via the slot, I didn't think about this.
In this case I don't think we need additional logic here :)